### PR TITLE
Feature/api score in validation result

### PR DESCRIPTION
**Pull request by thomas-siegrist**

### CODE REVIEW

**Comment by thomas-siegrist** [comment]
@chbuehlmann: can you do a review?

File: server/zally-server/src/main/java/org/zalando/zally/apireview/Score.kt [hunk 1]
```diff
@@ -0,0 +1,56 @@
+package org.zalando.zally.apireview
+
+import org.zalando.zally.rule.api.Severity
+import java.math.BigDecimal
+import java.math.RoundingMode
+import java.util.EnumMap
+import kotlin.math.min
+
+class Score {
+    companion object {
```
Comment (chbuehlmann) [inline server/zally-server/src/main/java/org/zalando/zally/apireview/Score.kt:10]: why static / as companion?

**Review by chbuehlmann — COMMENTED** [review]
mostly OK, see comment.

File: server/zally-server/src/main/java/org/zalando/zally/apireview/Score.kt (hunk 1 above)
Comment (thomas-siegrist) [reply server/zally-server/src/main/java/org/zalando/zally/apireview/Score.kt:10]: it's only a function, not an object with a value -> do you suggest to rewrite it to oo-style. would be possible

**Pull request merged**